src/components/search/content-highlights/data/hooks.js

```diff
@@ -43,6 +46,38 @@ export const useContentHighlights = (enterpriseUUID) => {
   };
 };
 
+export const useEnterpriseCuration = (enterpriseUUID) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [enterpriseCuration, setEnterpriseCuration] = useState([]);
+  const [fetchError, setFetchError] = useState();
+
+  useEffect(() => {
+    const fetchEnterpriseCuration = async () => {
+      try {
+        setIsLoading(true);
+        const response = await getEnterpriseCuration(enterpriseUUID);
+        const results = camelCaseObject(response.data.results);
+        const enterpriseCurationConfig = results[0];
+        setEnterpriseCuration(enterpriseCurationConfig);
```

Comment:
nit/curious: the default state for `enterpriseCuration` is an empty array, but here you're setting it to the first element (object) within the array. Should the initial state value be initialized as `undefined` (i.e., `useState()`) or an empty object instead to match the data type it ultimately receives?